geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/GetStackTracesFunction.java

```diff
@@ -36,7 +36,7 @@ public void execute(FunctionContext context) {
       if (memberNameOrId == null) {
         memberNameOrId = cache.getDistributedSystem().getDistributedMember().getId();
       }
-      Date timestamp = new Date();
+      Instant timestamp = Instant.now();
```

Comment:
This variable is redundant and not needed, you can just use `Instant.now()` when constructing the `StackTracesPerMember` instance.